docs/ref/request-response.txt

```diff
@@ -742,6 +746,13 @@ Attributes
 
     A bytestring representing the content, encoded from a string if necessary.
 
+.. attribute:: HttpResponse.headers
+
+    .. versionadded:: 3.2
+
+    A case insensitive, dict-like object that provides an interface to all
+    HTTP headers on the response.
```

Comment:
Add link to the above "Setting response headers" section?

Reply:
Cheers, done.